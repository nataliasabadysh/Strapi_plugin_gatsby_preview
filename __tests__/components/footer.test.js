import React from "react"
import renderer from "react-test-renderer"
import Footer from "../../src/components/footer"

// Sample test adapted from https://www.gatsbyjs.com/docs/how-to/testing/unit-testing/
describe("Footer", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})