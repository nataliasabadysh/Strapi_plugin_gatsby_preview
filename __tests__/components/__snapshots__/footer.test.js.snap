// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  id="footer"
>
  <div
    className="home-footer"
    id="home-footer"
  >
    <div
      className="my-5 by-3"
    >
      <div
        className="row"
      >
        <div
          className="col col-12 col-md-4"
        >
          <div
            className="text-center text-md-left"
            id="footer-logo"
          >
            <a
              href="/"
            >
              <img
                alt="BC Parks wordmark"
                src="test-file-stub"
              />
            </a>
          </div>
        </div>
        <div
          className="col col-12 col-md-8"
        >
          <div
            className="row"
          >
            <div
              className="col col-12 col-sm-4"
            >
              <div>
                <ul
                  className="footer-menu-list list-unstyled pt-3 pl-1 ml-3 ml-md-0 text-white"
                >
                  <li>
                    <div
                      className="font-weight-bold"
                    >
                      Get a permit
                    </div>
                    <div
                      className="footer-menu-divider"
                    />
                  </li>
                  <li>
                    <div
                      className="mt-2"
                    >
                      <a
                        href="/commercial-use/"
                      >
                        Commercial use
                      </a>
                    </div>
                  </li>
                  <li>
                    <div
                      className="mt-2"
                    >
                      <a
                        href="/commercial-use/filming-in-parks/"
                      >
                        Filming in parks
                      </a>
                    </div>
                  </li>
                  <li>
                    <div
                      className="mt-2"
                    >
                      <a
                        href="/commercial-use/travel-trade/"
                      >
                        Travel trade
                      </a>
                    </div>
                  </li>
                </ul>
              </div>
            </div>
            <div
              className="col col-12 col-sm-4"
            >
              <div>
                <ul
                  className="footer-menu-list list-unstyled pt-3 pl-1 ml-3 ml-md-0 text-white"
                >
                  <li>
                    <div
                      className="font-weight-bold"
                    >
                      Get involved
                    </div>
                    <div
                      className="footer-menu-divider"
                    />
                  </li>
                  <li>
                    <div
                      className="mt-2"
                    >
                      <a
                        href="/get-involved/donate/"
                      >
                        Donate
                      </a>
                    </div>
                  </li>
                  <li>
                    <div
                      className="mt-2"
                    >
                      <a
                        href="/get-involved/buy-licence-plate/"
                      >
                        Buy a licence plate
                      </a>
                    </div>
                  </li>
                  <li>
                    <div
                      className="mt-2"
                    >
                      <a
                        href="/get-involved/volunteer/"
                      >
                        Volunteer
                      </a>
                    </div>
                  </li>
                </ul>
              </div>
            </div>
            <div
              className="col col-12 col-sm-4"
            >
              <div>
                <ul
                  className="footer-menu-list list-unstyled pt-3 pl-1 ml-3 ml-md-0 text-white"
                >
                  <li>
                    <div
                      className="font-weight-bold"
                    >
                      Stay connected
                    </div>
                    <div
                      className="footer-menu-divider"
                    />
                  </li>
                  <li>
                    <div
                      className="mt-2"
                    >
                      <a
                        href="/contact/"
                      >
                        Contact us
                      </a>
                    </div>
                  </li>
                  <li>
                    <div
                      className="mt-2"
                    >
                      <a
                        href="https://engage.gov.bc.ca/bcparksblog/"
                      >
                        BC Parks blog
                      </a>
                    </div>
                  </li>
                  <li>
                    <div
                      className="d-inline-block mt-3"
                    >
                      <a
                        href="https://www.facebook.com/YourBCParks/"
                      >
                        <img
                          alt="Facebook"
                          src="test-file-stub"
                        />
                      </a>
                    </div>
                    <div
                      className="d-inline-block mt-3 ml-3"
                    >
                      <a
                        href="https://www.instagram.com/yourbcparks"
                      >
                        <img
                          alt="Instagram"
                          src="test-file-stub"
                        />
                      </a>
                    </div>
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="text-left text-sm-center py-3 mx-3 mx-md-0 mt-5 border-top border-white"
      >
        <div
          className="footer-utility-link d-inline-block"
        >
          <a
            href="/sitemap/"
          >
            Sitemap
          </a>
        </div>
        <div
          className="footer-utility-link d-inline-block"
        >
          <a
            href="https://www2.gov.bc.ca/gov/content/home/disclaimer"
          >
            Disclaimer
          </a>
        </div>
        <div
          className="footer-utility-link d-inline-block"
        >
          <a
            href="https://www2.gov.bc.ca/gov/content/home/privacy"
          >
            Privacy
          </a>
        </div>
        <div
          className="footer-utility-link d-inline-block"
        >
          <a
            href="https://www2.gov.bc.ca/gov/content/home/accessible-government"
          >
            Accessibility
          </a>
        </div>
        <div
          className="footer-utility-link d-inline-block"
        >
          <a
            href="https://www2.gov.bc.ca/gov/content/home/copyright"
          >
            Copyright
          </a>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
